dc3.py

```diff
@@ -0,0 +1,22 @@
+print ("Hello")
+symbol = input ("Jakie zadanie matematyczne chcesz wykonać? Dozwolone znaki to:\n +\n -\n *\n / \n\n")
```

Comment:
aby twój skrypt był jeszcze lepszy zastosowałbym na zmiennej `symbol` strip() -> co to robi? usuwa spacje z lewej i prawej

  ```suggestion
symbol = input ("Jakie zadanie matematyczne chcesz wykonać? Dozwolone znaki to:\n +\n -\n *\n / \n\n").strip()
```